clang/test/Driver/sycl-offload-intelfpga.cpp

```diff
@@ -243,14 +243,15 @@
 // CHK-FPGA-REPORT-OPT: aoc{{.*}} "-sycl" {{.*}} "-output-report-folder=[[OUTDIR]]{{/|\\\\}}file.prj"
 
 /// -fintelfpga output report file from dir/source
+/// check dependency file from dir/source
```

Comment:
Could the "first input name is the stem" notion also be tested?

Reply:
Sure, I can add a test to check the first input.  As for your `Xsycl-target-backend` question, I think that is automatically handled.  Any option that is passed via `-Xsycl-target-backend` is added to the end of the `aoc` call, so it will effectively take precedence.

Reply:
Just for curiosity, what does -Xsycl-target-backend? Does that mean if user use `dpcpp .... -fintelfpga -o A -Xso B then B takes precedence over A?

Reply:
`-Xsycl-target-backend` is longhand for `-Xs<opt>`.  Any of the options passed through will always be added to the end of the AOT tool call.